README.md

```diff
@@ -458,7 +464,23 @@ To use WSS flow on the egde and between ECC, do the following:
 
 **Changes in DataApp**
 
-In config.properties file
+In `application.properties` file:
+
+```
+application.verifyCheckSum=true
+```
+If the size of files for transfer are larger than 30Mb, please also modify the next properties in `application.properties` file:
+
+
+```
+#SFTP settings
+application.sftp.host=localhost
+application.sftp.port=2222
+application.sftp.username=test
```

Comment:
Good catch!

As previously agreed, I changed in code and property file to application.sftp.connectorId, but I forgot to change it here in README.md, it will be updated.